### PR TITLE
Gallery 01 03 SDK

### PR DESCRIPTION
**Pull request by kangsun-ctrl**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

Swagger PR these changes are based on: https://github.com/Azure/azure-rest-api-specs/pull/17985

### CODE REVIEW

**Review by fengzhou-msft — APPROVED** [review]

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/CommunityGalleryImage.cs [hunk 1]
```diff
@@ -56,7 +56,9 @@ public CommunityGalleryImage()
...
+        /// <param name="architecture">Possible values include: 'x64',
+        /// 'Arm64'</param>
+        public CommunityGalleryImage(OperatingSystemTypes osType, OperatingSystemStateTypes osState, GalleryImageIdentifier identifier, string name = default(string), string location = default(string), string type = default(string), string uniqueId = default(string), System.DateTime? endOfLifeDate = default(System.DateTime?), RecommendedMachineConfiguration recommended = default(RecommendedMachineConfiguration), Disallowed disallowed = default(Disallowed), string hyperVGeneration = default(string), IList<GalleryImageFeature> features = default(IList<GalleryImageFeature>), ImagePurchasePlan purchasePlan = default(ImagePurchasePlan), string architecture = default(string))
```
Comment (Sandido) [inline sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/CommunityGalleryImage.cs:61]: No need for a customization.

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/CommunityGalleryImageVersion.cs [hunk 2]
```diff
@@ -46,11 +46,18 @@ public CommunityGalleryImageVersion()
...
         /// decommissioning purposes. This property is updatable.</param>
-        public CommunityGalleryImageVersion(string name = default(string), string location = default(string), string type = default(string), string uniqueId = default(string), System.DateTime? publishedDate = default(System.DateTime?), System.DateTime? endOfLifeDate = default(System.DateTime?))
+        /// <param name="excludeFromLatest">If set to true, Virtual Machines
+        /// deployed from the latest version of the Image Definition won't use
+        /// this Image Version.</param>
+        /// <param name="storageProfile">Describes the storage profile of the
+        /// image version.</param>
+        public CommunityGalleryImageVersion(string name = default(string), string location = default(string), string type = default(string), string uniqueId = default(string), System.DateTime? publishedDate = default(System.DateTime?), System.DateTime? endOfLifeDate = default(System.DateTime?), bool? excludeFromLatest = default(bool?), SharedGalleryImageVersionStorageProfile storageProfile = default(SharedGalleryImageVersionStorageProfile))
```
Comment (Sandido) [inline sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/CommunityGalleryImageVersion.cs:54]: No need for customization.

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/SharedGalleryImage.cs [hunk 3]
```diff
@@ -55,7 +55,9 @@ public SharedGalleryImage()
...
+        /// <param name="architecture">Possible values include: 'x64',
+        /// 'Arm64'</param>
+        public SharedGalleryImage(OperatingSystemTypes osType, OperatingSystemStateTypes osState, GalleryImageIdentifier identifier, string name = default(string), string location = default(string), string uniqueId = default(string), System.DateTime? endOfLifeDate = default(System.DateTime?), RecommendedMachineConfiguration recommended = default(RecommendedMachineConfiguration), Disallowed disallowed = default(Disallowed), string hyperVGeneration = default(string), IList<GalleryImageFeature> features = default(IList<GalleryImageFeature>), ImagePurchasePlan purchasePlan = default(ImagePurchasePlan), string architecture = default(string))
```
Comment (Sandido) [inline sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/SharedGalleryImage.cs:60]: no need for customization

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/SharingProfileGroupTypes.cs [hunk 4]
```diff
@@ -18,6 +18,5 @@ public static class SharingProfileGroupTypes
     {
         public const string Subscriptions = "Subscriptions";
         public const string AADTenants = "AADTenants";
-        public const string Community = "Community";
```
Comment (Sandido) [inline sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/SharingProfileGroupTypes.cs:21]: Is this expected?

**Review by Sandido — CHANGES_REQUESTED** [review]

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/SharingProfileGroupTypes.cs (hunk 4 above)
Comment (Sandido) [reply sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/SharingProfileGroupTypes.cs:21]: based on this change, https://github.com/Azure/azure-rest-api-specs/pull/17985/files#diff-2861a3a31f910fbcb3cbc0af6f313527001ff65e8360cfcc566ae280a27b8bf9L1849 , looks like it is.

**Review by Sandido — APPROVED** [review]

**Pull request merged**